### PR TITLE
tst2-updates

### PR DESCRIPTION
**Pull request by scott062**
Dual Merge:
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/741)

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -47,16 +48,20 @@ export const init = (config) => {
     headers.tmusrname = localStorage.getItem('tmusrname');
   }
 
-  if (auth) {
-    renderLoading();
+  renderLoading();
```
Comment (elizabeth-jimenez) [inline src/index.jsx:51]: 😻

**Comment by elizabeth-jimenez** [comment]
@scott062 merged dev, git pull alert

File: src/index.jsx [hunk 2]
```diff
@@ -47,16 +59,32 @@ export const init = (config) => {
     headers.tmusrname = localStorage.getItem('tmusrname');
   }
 
-  if (auth) {
-    renderLoading();
```
Comment (elizabeth-jimenez) [inline src/index.jsx:51]: 👀

Comment (elizabeth-jimenez) [reply src/index.jsx:51]: teehee

**Review by elizabeth-jimenez — APPROVED** [review]
🐱

File: src/index.jsx [hunk 3]
```diff
@@ -3,60 +3,99 @@ import 'core-js/shim'; // included < Stage 4 proposals
...
+  const env = determineEnv(window.location.hostname);
+  const isPublic = includes(window.location.hostname, 'msappproxy');
+
   const auth = get(config, 'hrAuthUrl');
+  const publicAuth = get(config, 'hrAuthUrlPublic');
 
   // Only pass tmusrname header if localhost or metaphase environment
-  const isDev = some(['localhost', 'metaphasedev'], el => includes(window.location.hostname, el));
-  const withCredentials = !isDev;
+  const isLocalDev = env.includes(['localhost', 'metaphasedev']);
```
Comment (scott062) [inline src/index.jsx:50]: @elizabeth-jimenez this was the problem - doesn't do what i thought it did. Hence the some(xxxxxx, includes()) logic i was replacing.

**Pull request merged**